docs/src/content/docs/install.md

```diff
@@ -67,3 +67,31 @@ Example:
 ```console
 docker run -d -v $(PWD)/config.yml:/opt/config.yml -p 9090:9090 chproxy-test -config /opt/config.yml
 ```
+
+### systemd service
+
+Example systemd service configuration:
+
+```
+[Unit]
+Description=chproxy
```

Comment:
```suggestion
Description=CHProxy
Documentation=https://www.chproxy.org/
```

Reply:
Thank you, added the documentation link

I looked for refererences to capitalisation of chproxy and found different versions, the GitHub landing page (readme) reads 'chproxy', at chproxy.org I see 'Chproxy' while some source/config comments uses 'CHProxy'?

Reply:
I've checked, and the correct spelling is `Chproxy`. Thanks for your time and contributions 🤗

Reply:
Okay, sounds good to me 🙂 Changed to Chproxy

Thank you for the review